stdlib/2and3/pdb.pyi

```diff
@@ -52,11 +74,179 @@ class Pdb(Cmd):
             stdout: Optional[IO[str]] = ...,
             skip: Optional[Iterable[str]] = ...,
         ) -> None: ...
-    # TODO: The run* and set_trace() methods are actually defined on bdb.Bdb, from which Pdb inherits.
-    # Move these methods there once we have a bdb stub.
-    def run(self, statement: str, globals: Optional[Dict[str, Any]] = ...,
-            locals: Optional[Dict[str, Any]] = ...) -> None: ...
-    def runeval(self, expression: str, globals: Optional[Dict[str, Any]] = ...,
-                locals: Optional[Dict[str, Any]] = ...) -> Any: ...
-    def runcall(self, func: Callable[..., _T], *args: Any, **kwds: Any) -> Optional[_T]: ...
-    def set_trace(self, frame: Optional[FrameType] = ...) -> None: ...
+    def forget(self) -> None: ...
+    def setup(self, f: Optional[FrameType], t: Optional[TracebackType]) -> None: ...
+    def execRcLines(self) -> None: ...
+    def bp_commands(self, frame: FrameType) -> bool: ...
+    def interaction(self, frame: Optional[FrameType], traceback: Optional[TracebackType]) -> None: ...
+    def displayhook(self, obj: object) -> None: ...
+    def handle_command_def(self, line: str) -> bool: ...
+    def defaultFile(self) -> str: ...
+    def lineinfo(self, identifier: str) -> Union[Tuple[None, None, None], Tuple[str, str, int]]: ...
+    def checkline(self, filename: str, lineno: int) -> int: ...
+    def _getval(self, arg: str) -> object: ...
+    def print_stack_trace(self) -> None: ...
+    def print_stack_entry(self, frame_lineno: Tuple[FrameType, int], prompt_prefix: str = ...) -> None: ...
+    def lookupmodule(self, filename: str) -> Optional[str]: ...
+    def _runscript(self, filename: str) -> None: ...
+    def do_commands(self, arg: str) -> Optional[bool]: ...
+    def do_break(self, arg: str, temporary: bool = ...) -> Optional[bool]: ...
+    def do_tbreak(self, arg: str) -> Optional[bool]: ...
+    def do_enable(self, arg: str) -> Optional[bool]: ...
+    def do_disable(self, arg: str) -> Optional[bool]: ...
+    def do_condition(self, arg: str) -> Optional[bool]: ...
+    def do_ignore(self, arg: str) -> Optional[bool]: ...
+    def do_clear(self, arg: str) -> Optional[bool]: ...
+    def do_where(self, arg: str) -> Optional[bool]: ...
+    def do_up(self, arg: str) -> Optional[bool]: ...
+    def do_down(self, arg: str) -> Optional[bool]: ...
+    def do_until(self, arg: str) -> Optional[bool]: ...
+    def do_step(self, arg: str) -> Optional[bool]: ...
```

Comment:
All the `do_` methods until here never return anything, and a few from here on down return `1` (probably as a relict of the pre-bool days). I'm guessing the cmd interface requires Optional[bool], so this stub seems fine.